client/src/js/samples/components/Create/Create.js

```diff
@@ -108,6 +108,7 @@ class CreateSample extends React.Component {
     };
 
     render() {
+        this.state.subtraction = getActiveSubtraction(this.props);
```

Comment:
This should not be in this commit/PR. Also see https://reactjs.org/docs/state-and-lifecycle.html#using-state-correctly